packages/react-devtools-shared/src/backend/console.js

```diff
@@ -176,10 +176,7 @@ export function patch({
                       current,
                       ((method: any): 'error' | 'warn'),
                       // Copy args before we mutate them (e.g. adding the component stack)
-                      alreadyHasComponentStack
-                        ? // Replace component stack with an empty string in case there's a string placeholder for it.
-                          [...args.slice(0, -1), '']
-                        : args.slice(),
+                      args.slice(),
```

Comment:
This caused problems for RN+`LogBox` which did not pass component stack in as an additional parameter but instead as part of the original string. In hindsight, it's better for us just to not be clever here and pass through the string as-is.

Reply:
Agreed. It made more sense when we wrapped the error message. Now that we truncate (and most error messages are truncated) it's not that "bad" to display too much information.